src/index.ts

```diff
@@ -64,5 +64,9 @@ export const configureMonacoTailwindcss: typeof import('monaco-tailwindcss').con
           disposables.pop()?.dispose();
         }
       },
+
+      setConfig(tailwindConfig) {
```

Comment:
i guess this is  a good way to interact with this 'thing'
but i think the method name is a bit unfortunate and can be missleading that its not about `MonacoTailwindcssOptions`

so how about `setTailwindConfig`

Reply:
Makes sense. Should the `config` property from `MonacoTailwindcss` then also be renamed to `tailwindConfig`?

Reply:
nice ^^